### PR TITLE
[TACHYON-1489] Add unittests for BlockContainerIdGenerator

### PR DESCRIPTION
**Pull request by gpang**
https://tachyon.atlassian.net/browse/TACHYON-1489

### CODE REVIEW

File: servers/src/test/java/tachyon/master/block/meta/BlockContainerIdGeneratorTest.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+import tachyon.proto.journal.Block;
+import tachyon.proto.journal.Journal;
+
+/**
+ * Unit tests for {@link BlockContainerIdGenerator}.
+ */
+public final class BlockContainerIdGeneratorTest {
+  /** A non-default, container id for testing. */
+  private static final long TEST_ID = 1234;
+  private BlockContainerIdGenerator mGenerator;
```
Comment (apc999) [inline servers/src/test/java/tachyon/master/block/meta/BlockContainerIdGeneratorTest.java]: don't we have a maximum value for container Id? if yes, shall we test it?

Comment (gpang) [inline servers/src/test/java/tachyon/master/block/meta/BlockContainerIdGeneratorTest.java]: That's a good question. There is a limit, when it is reached, but it is not obvious what we should do about it, and where it should be handled. It is not being handled in the BlockContainerIdGenerator class currently.

**Comment by gpang** [comment]
For error handling, I created a JIRA to investigate how and where to handle the container id limit.
https://tachyon.atlassian.net/browse/TACHYON-1545

**Comment by apc999** [comment]
+1 LGTM

**Pull request merged**